Documentation/forms-intro.html

```diff
@@ -1,283 +1,598 @@
+          When the user enters the data, it is stored in the web as a triple
+          with this property as its predicate.
+        </td>
+      </tr>
+
+      <tr>
+        <td>default</td>
+
+        <td>[according to field type] Optional</td>
+
+        <td>
+          The input control is set to this value by default. It is easiest for
+          the user to enter this value. (This value is <b>not</b> automatically
+          stored by the form system if the user does not select or enter it in
+          some way.
+        </td>
+      </tr>
+    </table>
+
+    Other properties are given for each field type.
+
+    <h4 class="class">Form</h4>
+
+    <p>
+      The form itself has a collection of fields. The <tt>parts</tt> property
+      gives an order list of the fields in each form.
+    </p>
+
+    <table class="properties">
+      <tr>
+        <td>parts</td>
+
+        <td>rdf:Collection (aka List, Array) of Field</td>
+
+        <td>The parts of the form in the order in which they are</td>
+      </tr>
+
+      <tr>
+        <td><i>part</i></td>
+
+        <td>Field (Obsolete)</td>
+
+        <td>
+          A field which is a part of the form or group. This property is
+          obsolete. Use parts.
+        </td>
+      </tr>
+    </table>
+
+    If you use the obsolete "part" method for listing the parts of a form, then
+    each field needs an additional property:
+
+    <table class="properties">
+      <tr>
+        <td>sequence</td>
+
+        <td>Integer</td>
+
+        <td>The parts of the form in the order in which they are</td>
+      </tr>
+    </table>
+
+    For each part, declare its type, and the extra data that type requires, as
+    below.
+
+    <h4 class="class">Group</h4>
+
+    <p>
+      Group is a field which is just a collection of other fields. It is in fact
+      interchangeable with Form.
+    </p>
+
+    <h3>Single Value fields - Numeric</h3>
+    These prompt the user for a single value. They typically take default
+    values, and min and max values.
+
+    <h4 class="class">BooleanField</h4>
+
+    <p>A checkbox on the form, stored an RDF boolean true or false value.</p>
+
+    <h4 class="class">TriStateField</h4>
+
+    <p>
+      A checkbox on the form, stores an RDF boolean true or false value, or no
+      value if the box is left in its third, blank state.
+    </p>
+
+    <h4 class="class">IntegerField</h4>
+
+    <p>An RDF integer value</p>
+
+    <h4 class="class">DecimalField</h4>
+
+    <p>An RDF decimal value. Useful for monetary amounts</p>
+
+    <h4 class="class">FloatField</h4>
+
+    <p>A floating point number</p>
+
+    <h3>Single Value fields: Special Types</h3>
+
+    <h4 class="class">ColorField</h4>
+
+    <p>
+      A color picker is used, and genertes a string which is a CSS_compatible
+      color in a string like <tt>#ffeebb</tt>
+    </p>
+
+    <h4 class="class">DateField</h4>
+
+    <p>
+      Uses a date picker on a good browser. Leaves an RDF date literal as is
```

Comment:
```suggestion
      Uses a date picker on a good browser. Leaves an RDF date literal as its
```
"good" should perhaps also be changed to a less-subjective, more-objective descriptor, like "recent", "{spec/version}-compliant", etc.